recipe/meta.yaml

```
@@ -24,11 +24,13 @@ requirements:
     - tbb-devel
     - muparser
     - boost-cpp
+    - python
```

Seems that the CI job failed at build stage when `cmake` was trying to locate `boost.python`.

```suggestion
    - python
    - py-boost
```